SmartDeviceLink/SDLUnsubscribeVehicleData.h

```diff
@@ -1,270 +1,385 @@
+
+/**
+ *  Convenience init for unsubscribing to all possible vehicle data items.
+ *
+ *  @param accelerationPedalPosition Unsubscribe to accelerationPedalPosition
+ *  @param airbagStatus Unsubscribe to airbagStatus
+ *  @param beltStatus Unsubscribe to beltStatus
+ *  @param bodyInformation Unsubscribe to bodyInformation
+ *  @param cloudAppVehicleID Unsubscribe to cloudAppVehicleID
+ *  @param clusterModeStatus Unsubscribe to clusterModeStatus
+ *  @param deviceStatus Unsubscribe to deviceStatus
+ *  @param driverBraking Unsubscribe to driverBraking
+ *  @param eCallInfo Unsubscribe to eCallInfo
+ *  @param electronicParkBrakeStatus Unsubscribe to electronicParkBrakeStatus
+ *  @param emergencyEvent Unsubscribe to emergencyEvent
+ *  @param engineOilLife Unsubscribe to engineOilLife
+ *  @param engineTorque Unsubscribe to engineTorque
+ *  @param externalTemperature Unsubscribe to externalTemperature
+ *  @param fuelLevel Unsubscribe to fuelLevel
+ *  @param fuelLevelState Unsubscribe to fuelLevelState
+ *  @param fuelRange Unsubscribe to fuelRange
+ *  @param gps Unsubscribe to gps
+ *  @param headLampStatus Unsubscribe to headLampStatus
+ *  @param instantFuelConsumption Unsubscribe to instantFuelConsumption
+ *  @param myKey Unsubscribe to myKey
+ *  @param odometer Unsubscribe to odometer
+ *  @param prndl Unsubscribe to prndl
+ *  @param rpm Unsubscribe to rpm
+ *  @param speed Unsubscribe to speed
+ *  @param steeringWheelAngle Unsubscribe to steeringWheelAngle
+ *  @param tirePressure Unsubscribe to tirePressure
+ *  @param turnSignal Unsubscribe to turnSignal
+ *  @param wiperStatus Unsubscribe to wiperStatus
+ *  @param handsOffSteering - handsOffSteering
+ *  @return A SDLUnsubscribeVehicleData object
+ */
+- (instancetype)initWithAccelerationPedalPosition:(BOOL)accelerationPedalPosition airbagStatus:(BOOL)airbagStatus beltStatus:(BOOL)beltStatus bodyInformation:(BOOL)bodyInformation  cloudAppVehicleID:(BOOL)cloudAppVehicleID clusterModeStatus:(BOOL)clusterModeStatus deviceStatus:(BOOL)deviceStatus driverBraking:(BOOL)driverBraking eCallInfo:(BOOL)eCallInfo electronicParkBrakeStatus:(BOOL)electronicParkBrakeStatus emergencyEvent:(BOOL)emergencyEvent engineOilLife:(BOOL)engineOilLife engineTorque:(BOOL)engineTorque externalTemperature:(BOOL)externalTemperature fuelLevel:(BOOL)fuelLevel fuelLevelState:(BOOL)fuelLevelState fuelRange:(BOOL)fuelRange gps:(BOOL)gps headLampStatus:(BOOL)headLampStatus instantFuelConsumption:(BOOL)instantFuelConsumption myKey:(BOOL)myKey odometer:(BOOL)odometer prndl:(BOOL)prndl rpm:(BOOL)rpm speed:(BOOL)speed steeringWheelAngle:(BOOL)steeringWheelAngle tirePressure:(BOOL)tirePressure turnSignal:(BOOL)turnSignal wiperStatus:(BOOL)wiperStatus handsOffSteering:(BOOL)handsOffSteering;
```

Comment:
See above about `NSNumber`s

Reply:
For now 2 new initializers added: a manually created one and a script generated. Please advice which one to use.